test/Template/RouteTemplateVariableMiddlewareTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-helpers for the canonical source repository
+ * @copyright Copyright (c) 2019 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-helpers/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace ZendTest\Expressive\Helper\Template;
+
+use PHPUnit\Framework\TestCase;
+use Prophecy\Argument;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Zend\Expressive\Helper\Template\RouteTemplateVariableMiddleware;
+use Zend\Expressive\Helper\Template\TemplateVariableContainer;
+use Zend\Expressive\Router\Route;
+use Zend\Expressive\Router\RouteResult;
+
+class RouteTemplateVariableMiddlewareTest extends TestCase
+{
+    public function setUp()
```

Comment:
`protected`?

Reply:
It can be either `protected` or `public`. We're not terribly consistent in the framework in terms of using one or the other. I typically define this and `tearDown()` as public.